drivers/overlay/overlay.go

```diff
@@ -353,44 +353,54 @@ func (d *Driver) Create(id, parent, mountLabel string, storageOpt map[string]str
 
 func (d *Driver) getLower(parent string) (string, error) {
 	parentDir := d.dir(parent)
-
 	// Ensure parent exists
 	if _, err := os.Lstat(parentDir); err != nil {
 		return "", err
 	}
-
 	// Read Parent link fileA
 	parentLink, err := ioutil.ReadFile(path.Join(parentDir, "link"))
 	if err != nil {
 		return "", err
 	}
 	lowers := []string{path.Join(linkDir, string(parentLink))}
-
 	parentLower, err := ioutil.ReadFile(path.Join(parentDir, lowerFile))
 	if err == nil {
 		parentLowers := strings.Split(string(parentLower), ":")
-		lowers = append(lowers, parentLowers...)
+		for _, p := range parentLowers {
+			lowers = append(lowers, p)
+		}
 	}
```

Comment:
I guess we don't need to do the d.dir(p) any longer, so I reverted this back to the original code.